File: pkg/doctor/testdata-packs/ewcs/negative.js
// already migrated: checkout session create carries no top-level amount,
// and amount on a non-matching resource must not fire either
await stripe.checkout.sessions.create({
  line_items: [{price_data: {currency: 'eur', unit_amount: 1099, product_data: {name: 'T'}}, quantity: 1}],
  mode: 'payment', ui_mode: 'elements', return_url: 'https://x/return',
});
await stripe.refunds.create({ amount: 500 });

File: pkg/doctor/testdata-packs/flex/negative.py
# beta param at the WRONG nesting (top-level) must not match,
# and final_capture on a non-capture resource must not match
import stripe
stripe.PaymentIntent.create(
    amount=5000,
    currency="usd",
    request_incremental_authorization_support=True,
)
config = {"final_capture": False}
